Aptoide/aptoide-client-v8
# [ASV-1162] [ASV-1177] Adapt curation api

**What does this PR do?**

   Changes the necessary fields in the requests and the responses related to the editorial e appc cards

**Database changed?**

    No

**Where should the reviewer start?**

-EditorialFragment.java
-EditorialService.java

**How should this be manually tested?**

-Check if APPC card appears (change the aptoide_uid in the layout=appc_card_info request) and if the know more and mark as read works correctly through the 7.20181019 endpoint.
Check if the EditorialCard appears in home (change the vercode in the getStoreWidgets request to 9400) and check if after pressing the editorial card in home it shows the editorial view and everything is working fine

**What are the relevant tickets?**

https://aptoide.atlassian.net/browse/ASV-1162
https://aptoide.atlassian.net/browse/ASV-1177


**Code Review Checklist**
